styles/profile_styles.css

```
@@ -60,8 +60,85 @@
 }
 #contact_info p {
     font-size: 1vw;
+}*/
+
+#contact_info {
+    height: fit-content;
+    grid-column: 7 / 9;
+    grid-row: 1 / 4;
+}
+
+/*.calendar {
+    grid-area: calendar;
+    display: grid;
+    grid-template-columns: repeat(7,1fr);
+    grid-template-rows: repeat(6,1fr);
+    justify-items: center;
+    align-items: center;  
+    width: 100% 
+}
+
+.calendar > div {
+    font-size: 10pt;
 }
 
+.day:not(:hover)::before {
+    content: ".";
+    font-size: 10pt;
+}
+
+.day span {
+    display: none;
+}
+
+.day:hover span {
+    display: inline;
+} */
+
+#calendar {
+    width: 100%;
+    grid-area: calendar;
+}
+
+table {
+    text-align: center;
+
+}
+
+tr {
+    height: auto;
+}
+
+tr > {
```

Wait does that selector.. work? `tr >` ? Would that just select the direct children of `tr` elements?

Seems a little suspect but if it works, it works. Can change for clarity:
```suggestion
tr > * {
```

---

Again, CSS suggestion untested, my version may not work.